src/structures/Client.ts

```diff
@@ -325,21 +337,47 @@ export class Client extends DJSClient
 		const [, rawId, info2]: [string, string, string] = exec as any;
 		const id: number | 'Manager' = rawId === 'Manager' ? rawId : parseInt(rawId);
 
-		/* eslint-disable-next-line no-cond-assign */
+		/* eslint-disable no-cond-assign */
+		// Discord requested a reconnect
+		if (exec = / *\[RECONNECT\] *(.+) */.exec(info2))
```

Comment:
```suggestion
		if (exec = /\s*\[RECONNECT\]\s*(.+)\s*/.exec(info2))
```

In modern RegEx engines, you should always use `\s` (tokens) instead of ` ` when dealing with quantifiers.
> This comment applies to all the lines below it.